File: backend/members-service/src/main/java/com/ringbox/members/web/MemberDto.java
package com.ringbox.members.web;

import java.math.BigDecimal;
import java.time.Instant;

public record MemberDto(
    Integer id,
    String fullName,
    String phone,
    String plan,
    String status,
    BigDecimal weightKg,
    String level,
    Instant createdAt) {}

File: backend/members-service/src/dto/memberDto.js
function toMemberDto(row) {
  return {
    id: row.id,
    fullName: row.full_name,
    phone: row.phone,
    plan: row.plan,
    status: row.status,
    weightKg: row.weight_kg,
    level: row.level,
    createdAt: row.created_at
  };
}

module.exports = { toMemberDto };
